vnext/Desktop/BeastWebSocketResource.h

```diff
@@ -3,6 +3,9 @@
 
 #pragma once
 
+#include <boost/beast/core/multi_buffer.hpp>
```

Comment:
>beast [](start = 16, length = 5)

So, just that I understand dependency.
Currently this file is being shipped with react-native-win32.dll, right?
We need it because although WebSocket has rolled out 100% to Production, we still want to allow possibility of rolling back, just in case something bad happens with WinRT WebSockets.

Let's say after couple of months where WinRT WebSockets prove as a stellar performer for Augloop, could we move this code out of react-native-win32.dll?

Reply:
Yes. This code will be removed from the DLL.

The upgrade will still be useful for in-process test networking servers.